src/api/comments.ts

```diff
@@ -0,0 +1,19 @@
+import { Comment } from '../types/Comment';
+import { client } from '../utils/fetchClient';
+
+export const getComments = (postId: number) => {
+  return client.get<Comment[]>(`/comments?postId=${postId}`);
+};
```

Comment:
Consider adding error handling for the `getComments` function to manage potential errors gracefully.